<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListTable</class>
 <widget class="QWidget" name="ListTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>55</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>55</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>55</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="text">
      <string>课程编号</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_id">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>3</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="text">
      <string>课程名称</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_name">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="text">
      <string>学分</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_credit">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="text">
      <string>学期数</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_semester">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="text">
      <string>时间安排</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_timeschedual">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="text">
      <string>上课地点</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_classroom">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
